src/Cleanup.php

```diff
@@ -110,11 +110,12 @@ public function preMigration(string $mainRoleName): void
             false,
         );
 
-        if ($mainRoleExistsOnTargetUser && !$hasMainRoleOwnership) {
-            throw new UserException('Main role exists but is not assigned to migrate user.');
-        }
-
         if (!$mainRoleExistsOnTargetUser) {
```

Comment:
@ondrajodas Moc jsem nepochopil (ani na slacku), proč se to teď má vyhazovat při `!$mainRoleExistsOnTargetUser` 🤔

Reply:
zkusím to trochu líp

Reply:
jo tak protože ta podmínka byla blbě... teď to je tak že:
pokud nemá migrační uživatel hlavní roli `$mainRoleExistsOnTargetUser` tak se stane:
1/ nemá na ní ownership tak to spadne
2/ má na ní ownership tak si jí přiřadí